layer/runtime_layer_data.cc

```diff
@@ -131,9 +135,20 @@ void RuntimeLayerData::LogAndRemoveQueryPools() {
       // FIXME: We should adjust the elapsed units to account for the current
       // GPU frequency. Calling vkGetPhysicalDeviceProperties here causes the
       // driver to crash, however.
-      Log("pipeline_execution", GetPipelineHash(info->pipeline),
-          absl::StrCat(timestamp1 - timestamp0, ",", invocations[0], ",",
-                       invocations[1]));
+      HashVector pipeline = GetPipelineHash(info->pipeline);
+      std::vector<int64_t> hashes(pipeline.begin(), pipeline.end());
+      RuntimeEvent event("pipeline_execution", hashes,
+                         DurationClock::duration(timestamp1 - timestamp0),
+                         invocations[0], invocations[1]);
+      LogEvent(&event);
+
+      std::stringstream pipeline_hash_str;
+      pipeline_hash_str << std::quoted(PipelineHashToString(pipeline));
```

Comment:
Hmm, since this pattern keeps coming up, maybe we should bring back the QuoteStr function and implement it in terms of `std::quoted`?

This can be done in a separate PR, no need to change this line

Reply:
This code will be removed in the future PRs because the logger will take care of the quoted strings.